README.md

```
@@ -1,19 +1,486 @@
+
+#### 2.2 AndroidManifest.xml — Application 등록
+
+`AndroidManifest.xml`에서 `android:name`에 `Application` 서브클래스를 지정합니다.
+
+```xml
+<application
+    android:name=".MyApplication"
+    ...>
+```
+
+            initKoin {
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)